akka-stream/src/main/scala/akka/stream/impl/io/SslTlsCipherActor.scala

```diff
@@ -432,6 +432,7 @@ private[akka] class SslTlsCipherActor(settings: ActorMaterializerSettings,
         currentSession = session
         corkUser = false
     }
+    outputBunch.enqueue(UserOut, SessionBytes(currentSession, ByteString.empty))
```

Comment:
This is not guaranteed to be met with enthusiasm by the downstream subscriber: before calling `handshakeFinished` we have already called `flushToUser` which means that we don’t know if there is any more demand outstanding at this point.